fullstack-test.sh

```
@@ -0,0 +1,135 @@
+
+  # Set solana config to localhost
+  solana config set --url localhost
+
+  # Wait for solana-test-validator to be ready
+  echo -ne "${YELLOW}Waiting for solana-test-validator to be ready...${NC}\r"
+  for i in {1..60}; do
+    if curl -s http://127.0.0.1:8899/health > /dev/null 2>&1; then
+      echo -e "\033[2K${GREEN}solana-test-validator is ready${NC}"
+      break
+    fi
+    if [ $i -eq 60 ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)